src/java/com/vaklinov/zcashui/Util.java

```diff
@@ -107,7 +107,17 @@ public static boolean arraysAreDifferent(String ar1[][], String ar2[][])
 		
 		return false;
 	}
-	
+
+	public static final char[] HEX_ARRAY = "0123456789abcdef".toCharArray();
+	public static String bytesToHex(byte[] bytes) {
+		char[] hexChars = new char[bytes.length * 2];
```

Comment:
indentation seems off here

Reply:
If you look at the code above, it seems right

Reply:
we should refactor IMO